### DISCUSSION TITLE
Is linkDir renaming files?

### DISCUSSION THREAD

**Discussion by Friday13th87** [cross-seed/cross-seed#586] [Q&A]
Hello!
My question is about "linkDir".
If i have 2 torrents which are basically the same when looking at the data but they have different namings for the individual files.
Will a data based search with linkDir create links with the names of the mached torrents?
If not, is there a way to automaticly rename either the linked files or the names in the torrent client (in my case qbittorrent)?

**Comment by zakkarry** [comment]
With risky, it will match differently named files and link them to their name in the target. This is riskier than safe, but removes the requirement of the name being the same. This is one reason why linking is done for data-based matching.
